src/fondant/compiler.py

```diff
@@ -459,10 +465,16 @@ def resolve_imports(self):
     @staticmethod
     def _set_configuration(task, fondant_component_operation):
         # Unpack optional specifications
+        cpu_limit = fondant_component_operation.cpu_limit
+        memory_limit = fondant_component_operation.memory_limit
         number_of_accelerators = fondant_component_operation.number_of_accelerators
         accelerator_name = fondant_component_operation.accelerator_name
 
         # Assign optional specification
+        if cpu_limit is not None:
```

Comment:
should we also add the cpu/memory request for Vertex?

Reply:
According to the [docs](https://cloud.google.com/vertex-ai/docs/pipelines/machine-types), it only supports limits.